gtsam/hybrid/tests/testDCMixtureFactor.cpp

```diff
@@ -21,48 +21,72 @@ using namespace gtsam;
 
 /******************************************************************************/
 
-/*
- * Test DCDiscreteFactor using a simple mixture.
- *
- * Construct a single factor (for a single variable) consisting of a
- * discrete-conditional mixture. Here we have a "null hypothesis" consisting of
- * a Gaussian with large variance and an "alternative hypothesis" consisting of
- * a Gaussian with smaller variance.
- */
-TEST(TestSuite, dcdiscrete_mixture) {
+// /*
```

Comment:
Commented out? Decide if in this PR or not :-) if not, leave it out.

Reply:
This is the last thing that needs to be done.